dascore/io/index/schema.py

```
@@ -283,6 +283,12 @@ def _columns(row_type: type[NamedTuple]) -> MappingProxyType[str, str]:
         "distance_max",
         "distance_step",
         # flat-relation (spool-facing) names
+        "source_path",
```

Ruff (0.16.1)

[warning] Sets should not contain duplicate item `"source_path"`

Remove duplicate item

(B033)